smithy-swift-codegen/src/test/kotlin/httpResponse/HttpResponseBindingIgnoreQuery.kt

```diff
@@ -16,7 +16,7 @@ import shouldSyntacticSanityCheck
 class HttpResponseBindingIgnoreQuery {
 
     @Test
-    fun `001 Output httpResponseBinding sets query to nil`() {
+    fun `001 Output httpResponseBinding ignores query trait & decodes field`() {
```

Comment:
The premise of this test is incorrect.  The member with `@httpQuery` trait should not be set to nil; it should be read from the response.

The name of the test & the expectation are adjusted accordingly.